api/lib/opentelemetry/distributed_context/propagation/http_text_format.rb

```diff
@@ -37,10 +42,11 @@ def extract(carrier)
         # @yield [Carrier, String, String] carrier, header key, header value.
         def inject(context, carrier)
           yield carrier, TraceParent::TRACE_PARENT_HEADER, TraceParent.from_context(context).to_s
+          yield carrier, TRACESTATE_HEADER, context.tracestate unless context.tracestate.nil?
         end
 
         def fields
-          [TraceParent::TRACE_PARENT_HEADER]
+          [TraceParent::TRACE_PARENT_HEADER, TRACESTATE_HEADER]
```

Comment:
Should we yak shave an allocation and make this array a constant?

Reply:
👍 shaved.